### ISSUE TITLE
Which model metric is being evaluated - on val_half or motchallenge?

### ISSUE THREAD

**Issue by QAiStudio** [inakierregueab/YOLO11-JDE#22]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, I noticed that in your code, you are using val_half to evaluate the model. So, I would like to clarify the metrics mentioned in the paper. Are you using val_half or motchallenge to measure the results?

def mot_eval(validator, period=1):
    is_train = validator.training   # Check if the model is being trained
    if is_train:
        if validator.epoch % period != 0 or validator.epoch == 1:
            return  # Evaluate only every 'period' epochs after the first epoch

    # TODO: DEFINE DATASET, externalize
    dataset_name = 'MOT17/val_half'
    seqmap_file = './tracker/evaluation/TrackEval/data/gt/mot_challenge/seqmaps/MOT17-val_half.txt'

**Comment by inakierregueab** [comment]
Hi, MOT17 is composed of train and test splits. Ground truth labels are only available for the training set. Since there is no validation set, a common approach is to construct a MOT17 validation set using the second half of each training sequence and chop off the videos in ETH that are overlapped with the MOT16 benchmark. This validation set is used to monitor training and perform ablations. The final model configuration is used to perform inference on the MOT17 test set. Evaluation on this split can only be performed by submitting results to the MOTChallenge webpage (https://motchallenge.net/results/MOT17/). It is a private evaluation benchmark.
